go.mod

```diff
@@ -90,6 +90,7 @@ require (
 	github.com/kr/pretty v0.3.1
 	github.com/mattn/go-isatty v0.0.19
 	github.com/microsoft/kiota-abstractions-go v1.2.0
+	github.com/microsoft/kiota-http-go v1.0.1
```

Comment:
How did updating to charm/v11 change this package to a direct dependency? I didn't see it used in charm/v11